ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/config/WorkflowConfig.scala

```diff
@@ -126,50 +126,44 @@ object WorkflowConfig {
 
     final case class TransactionsStreamConfig(
         name: String,
-        filters: List[PartyFilter],
-        protected val partyNamePrefixFiltersO: Option[List[PartyNamePrefixFilter]] = None,
+        filters: List[PartyFilter] = List.empty,
+        partyNamePrefixFilters: List[PartyNamePrefixFilter] = List.empty,
         beginOffset: Option[LedgerOffset] = None,
         endOffset: Option[LedgerOffset] = None,
         objectives: Option[StreamConfig.TransactionObjectives] = None,
         subscriptionDelay: Option[FiniteDuration] = None,
         override val maxItemCount: Option[Long] = None,
-        override val timeoutInSecondsO: Option[Long] = None,
+        override val timeoutDurationO: Option[FiniteDuration] = None,
```

Comment:
Would it be actually better by just calling it `timeout` ? or `cancelStreamTimeout`

Reply:
Renamed to `timeoutO`